airflow/executors/base_executor.py

```diff
@@ -128,6 +128,7 @@ def queue_task_instance(
             pool=pool,
             pickle_id=pickle_id,
             cfg_path=cfg_path,
+            interactive=self.__class__.__name__ == "KubernetesExecutor",
```

Comment:
Given [AIP-51](https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-51+Removing+Executor+Coupling+from+Core+Airflow), can we tweak how this check is implemented? Maybe have a `supports_interactive` attribute on BaseExecutor? It just seems weird to require a base class to have knowledge about subclasses.

Reply:
+1 to @blag 's suggestion

Reply:
or... "requires interactive"